lib/DDG/Spice/BrainyQuote.pm

```diff
@@ -26,6 +26,10 @@ handle remainder => sub {
     if(/^\w\.\w/) {
         s/\./\. /g;
     }
+    
+    # Hacky fix to avoid triggering on 'stock quote'
+    # 'stock quote' is handled by the Stocks IA
+    s/stock//g;
```

Comment:
I guess I wasn't very clear, but this won't catch the case where the query is "apple stock quote" or similar.

So what we really want to do is check if the original query contains the word "stock quote" and if so, bail out with a `return;`

You can use the `$req` object which contains several useful properties, including `$req->query` which provides the original query and `$req->query_lc` which provides the lower-cased query.

So I suggest you run a regex against that for `m/stock quote/` and `return;` if it's true.